nncf/experimental/openvino_native/quantization/algorithms/fast_bias_correction/ov_backend.py

```diff
@@ -0,0 +1,167 @@
+from nncf.experimental.openvino_native.graph.transformations.commands import OVTargetPoint
+from nncf.experimental.openvino_native.statistics.collectors import OVMeanStatisticCollector
+from nncf.experimental.openvino_native.statistics.collectors import OVNNCFCollectorTensorProcessor
+from nncf.experimental.openvino_native.tensor import OVNNCFTensor
+from nncf.quantization.algorithms.fast_bias_correction.backend import ALGO_BACKENDS
+from nncf.quantization.algorithms.fast_bias_correction.backend import FastBiasCorrectionAlgoBackend
+
+
+@ALGO_BACKENDS.register(BackendType.OPENVINO)
+class OVFastBiasCorrectionAlgoBackend(FastBiasCorrectionAlgoBackend):
+
+    @property
+    def layers_with_bias_metatypes(self) -> List[OVOpMetatype]:
+        return LAYERS_WITH_BIAS_METATYPES
+
+    @property
+    def channel_axis_by_types(self) -> Dict[OVOpMetatype, int]:
+        return {
+            OVConvolutionMetatype: 1,
+            OVConvolutionBackpropDataMetatype: 1,
+            OVMatMulMetatype: -1
+        }
+
+    @property
+    def tensor_processor(self) -> OVNNCFCollectorTensorProcessor:
+        return OVNNCFCollectorTensorProcessor()
+
+    @staticmethod
+    def model_transformer(model: ov.Model) -> OVModelTransformer:
+        return OVModelTransformer(model)
+
+    @staticmethod
+    def target_point(target_type: TargetType, target_node_name: str, port_id: int) -> OVTargetPoint:
+        return OVTargetPoint(target_type, target_node_name, port_id)
+
+    @staticmethod
+    def bias_correction_command(target_point: OVTargetPoint, bias_value: np.ndarray) -> OVBiasCorrectionCommand:
+        return OVBiasCorrectionCommand(target_point, bias_value)
+
+    @staticmethod
+    def model_extraction_command(inputs: List[str], outputs: List[str]) -> OVModelExtractionCommand:
+        return OVModelExtractionCommand(inputs, outputs)
+
+    @staticmethod
+    def mean_statistic_collector(reduction_shape:
+                                 ReductionShape,
+                                 num_samples: int = None,
+                                 window_size: int = None) -> OVMeanStatisticCollector:
+        return OVMeanStatisticCollector(reduction_shape, num_samples, window_size)
+
+    @staticmethod
+    def get_input_output_names(node: NNCFNode, nncf_graph: NNCFGraph) -> Tuple[str, str]:
+        input_name = node.node_name
+        output_name = node.node_name
+
+        bias_node = OVFastBiasCorrectionAlgoBackend.get_bias_node(node, nncf_graph)
+        if bias_node is not None:
+            output_name = bias_node.node_name
+        return input_name, output_name
+
+    @staticmethod
+    def create_blob(shape: Tuple[int], data: List[float]) -> np.ndarray:
+        blob = np.zeros(shape)
+        for i, value in enumerate(data):
+            blob[:, i] = value
+        blob = blob.astype(np.float32)
+        return blob
+
+    @staticmethod
+    def get_bias_value(model: ov.Model, bias_node: NNCFNode) -> np.ndarray:
```

Comment:
Could you please move this method's body to the `nncf/experimental/openvino_native/graph/node_utils.py` module as a separate method?